orders/src/main/java/com/shelach/orders/controllers/OrdersController.java

```diff
@@ -26,8 +30,10 @@ public String mainPage(ExtendedModelMap model) {
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
         String username = authentication.getName();
         model.put("username", username);
-        model.put("products", productsService.getProducts(username));
+        List<Order> orders = fetchOrdersService.getProducts(username);
         log.info("Orders page ready for user <{}>", username);
+        OrderList orderList = new OrderList(orders);
+        model.addAttribute("orders", orderList);
```

Comment:
You changed the name here, my HTML page uses "products", need to adjust

Reply:
I'll adjust your  html once I rebase